### PR TITLE
Feature html

### PR DESCRIPTION
**Pull request by bruce-gordon**
#### What does this PR do?
Provides a page layout matching Iteration 0 design, with corresponding CSS styling.  
Also added Activity class to activity.js.

#### Where should the reviewer start?
Open index.html, styles.css, and activity.js

#### How should this be manually tested?
Open index.html in web browser.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,57 @@
...
+              <button class="start-button">START ACTIVITY</button>
+            </div>
+          </div>
+        </section>
+      </section>
+      <section class="past-activities-section">
+        <div class="past-activities-title">
+          Past Activities
+        </div>
+        <div class="logged-activities">
```
Comment (Hdemaceo) [inline index.html:49]: In the Iteration 0 rubric, these sentences are _slightly_ indented more than what we currently have.

File: styles.css [hunk 2]
```diff
@@ -1 +1,149 @@
+/* Main background: #2B2733
+Text on main background: #CBC9CF
+Header, cards: #46424D
+Card shadow: #26222D
+Text on header and cards: #FFF
+Study: #B3FD78
+Meditate: #C278FD
+Exercise: #FD8078
+Error: #EFB7EC */
+/* separate input lines between minutes and seconds
```
Comment (Hdemaceo) [inline styles.css:10]: Lets get rid of lines 10 & 11 in our styles.css

**Review by Hdemaceo — COMMENTED** [review]
Solid! Iteration 0 & 1 complete.

File: index.html (hunk 1 above)
Comment (bruce-gordon) [reply index.html:49]: Great catch!  We can definitely fix that on the next branch.

**Pull request merged**